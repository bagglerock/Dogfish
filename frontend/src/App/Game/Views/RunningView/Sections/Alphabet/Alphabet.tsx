import { Card } from 'app/share/Card';
import { map, split } from 'lodash';
import React from 'react';

export const Alphabet: React.FC = () => {
  const alphabet = 'abcdefghijklmonpqrstuvwxyz';

  return (
    <div className="d-flex flex-wrap game-card m-2">
      {map(split(alphabet, ''), letter => {
        return (
          <Card key={letter} onClick={() => console.log(letter)}>
            <p className="p-style-1">{letter}</p>
          </Card>
        );
      })}
    </div>
  );
};
