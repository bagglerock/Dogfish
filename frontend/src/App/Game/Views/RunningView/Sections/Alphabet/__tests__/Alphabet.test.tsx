import { render } from '@testing-library/react';
import { Alphabet } from 'app/Game/Views/RunningView/Sections/Alphabet/Alphabet';

test('renders Alphabet', () => {
  const { container } = render(<Alphabet updateWord={jest.fn()} />);

  expect(container).toMatchSnapshot();
});
