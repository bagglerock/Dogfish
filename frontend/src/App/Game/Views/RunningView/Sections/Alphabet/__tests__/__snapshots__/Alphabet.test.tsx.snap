// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Alphabet 1`] = `
<div>
  <div
    class="d-flex flex-wrap game-card m-2"
  >
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        A
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        B
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        C
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        D
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        E
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        F
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        G
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        H
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        I
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        J
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        K
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        L
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        M
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        N
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        O
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        P
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        Q
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        R
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        S
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        T
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        U
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        V
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        W
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        X
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        Y
      </p>
    </div>
    <div
      class="game-card m-2"
    >
      <p
        class="p-style-1"
      >
        Z
      </p>
    </div>
  </div>
</div>
`;
